tests/scripts/list-identifiers.sh

```diff
@@ -41,7 +41,7 @@ rm -f identifiers
 
 grep '^[^ /#{]' $HEADERS | \
     sed -e 's/^[^:]*://' | \
-    egrep -v '^(extern "C"|(typedef )?(struct|enum)( {)?$|};?$)' \
+    egrep -v '^(extern "C"|(typedef )?(struct|union|enum)( {)?$|};?$)' \
```

Comment:
Dear Gilles,

Thanks for letting Mr. Check Names know about this essential keyword.

Sincerely,
Potato man